core/src/main/java/io/questdb/cairo/sql/RecordCursorFactory.java

```diff
@@ -58,6 +58,11 @@
  * }
  */
 public interface RecordCursorFactory extends Closeable, Sinkable, Plannable {
+
+    int SCAN_DIRECTION_BACKWARD = 2;
```

Comment:
nit: I'd name things with the TIMESTAMP prefix, e.g. 
```
TIMESTAMP_SCAN_DIRECTION_BACKWARD
TIMESTAMP_SCAN_DIRECTION_FORWARD
TIMESTAMP_SCAN_DIRECTION_OTHER
```

Reply:
I'd rather avoid connecting direction with timestamp too much because it's perfectly valid for select on table without designated timestamp to show e.g. forward scan . 
Sure, we mostly use direction when also checking for timestamp order, but in my view scan direction is more generic piece of information .

Reply:
getScanDirection() alone doesn't say if scan is in timestamp order . All call sites have to check if factory has timestamp idx first . If we change method to getTimestampScanDirection() then shouldn't it return _OTHER value for heap tables (those without designated timestamp) ?
I do agree that getScanDirection()'s meaning is fuzzy for Sorted factories though .